### PR TITLE
Feature#12 register symptoms

### LINKED ISSUE #12: Register symptoms
# REQUIREMENTS
We are going to do the register symptoms screen UI

# DESIGN
COVID ALERTS V2

# Acceptance Criteria
List all required acceptance criteria
- [ ] Nice UI
- [ ] Code fixed with prettier
- [ ] Navigation still working

### PR DESCRIPTION
**Pull request by vmagana3**
# Description
We implement register symptoms UI to check users recent  symptoms.
We use React Native elements library to use checkbox element

  
  ## Screenshots 
image

### CODE REVIEW

**Review by JacintoOrtiz15745 — DISMISSED** [review]
Looks good!

**Review by JorgeGarciaRa — DISMISSED** [review]
GOOD JOB.

**Review by ChantalRamirez — DISMISSED** [review]
Good job

**Review by Joaquin-Niembro — DISMISSED** [review]

File: COVID19/src/screens/registerSymptoms.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    id: '10',
+    name: 'Sore throat',
+  },
+];
+
+const RegisterSymptoms = () => {
+  return (
+    <View style={styles.supermainContainer}>
+      <View style={styles.mainContainer}>
+        <Text style={styles.title}>What symptoms do you have?</Text>
```
Comment (fernandoamz) [inline COVID19/src/screens/registerSymptoms.js:55]: Add a file constant with all texts

File: COVID19/src/utils/functions/mainFunctions.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+const getFixedDate = () => {
+    let date = new Date();
+    let day = date.getDate();
+    let month = date.getMonth() + 1;
+    let year = date.getFullYear();
+    let fullDate = '';
+    if (month < 10) {
+      fullDate = `${day}/0${month}/${year}`;
```
Comment (fernandoamz) [inline COVID19/src/utils/functions/mainFunctions.js:8]: You can make a function to format this value and you can recycle

File: COVID19/src/utils/functions/mainFunctions.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      today.getHours() + ':' + today.getMinutes() + ':' + today.getSeconds();
+    return time;
+  };
+
+  export const MainFunctions = {
+    getFixedDate,
+    getFixedTime,
+  };
+ 
+
```
Comment (fernandoamz) [inline COVID19/src/utils/functions/mainFunctions.js:27]: New blank line here

File: COVID19/src/components/CustomCheck.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import React, {useState} from 'react';
+import {CheckBox} from 'react-native-elements';
+import styles from '../utils/styles/symptomsStyles/symptomsStyles';
+
+const CustomCheck = ({name}) => {
+  const [check, isCheck] = useState(false);
+  return (
+    <CheckBox
+      checkedColor="#5A4DCC"
+      uncheckedColor="#5A4DCC"
```
Comment (fernandoamz) [inline COVID19/src/components/CustomCheck.js:10]: Use the colors constant

File: COVID19/src/components/CustomCheck.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
+import React, {useState} from 'react';
+import {CheckBox} from 'react-native-elements';
+import styles from '../utils/styles/symptomsStyles/symptomsStyles';
+
+const CustomCheck = ({name}) => {
+  const [check, isCheck] = useState(false);
+  return (
```
Comment (fernandoamz) [inline COVID19/src/components/CustomCheck.js:7]:
```suggestion
  const [check, isCheck] = useState(false);
  
  return (
```

File: COVID19/src/screens/main.js [hunk 6]
```diff
@@ -1,29 +1,50 @@
...
         <View style={styles.optionsUp}>
-          <ButtonMain text='Register Symptoms' img={require('../utils/images/symptoms.png')} onP={()=>{navigation.navigate('Login')}}/>
-          <ButtonMain text='Register Labs' img={require('../utils/images/Labs.png')} onP={()=>{navigation.navigate('Login')}}/>
-        </View>
-        <View style={styles.optionsDown}>
-          <ButtonMain text='See Locations' img={require('../utils/images/locations.png')} onP={()=>{navigation.navigate('Login')}}/>
+          <ButtonMain
+            text="Register Symptoms"
+            img={require('../utils/images/symptoms.png')}
+            onP={() => {
```
Comment (fernandoamz) [inline COVID19/src/screens/main.js:25]: You should use a better name for this prop.

File: COVID19/src/screens/main.js [hunk 7]
```diff
@@ -1,29 +1,50 @@
...
+          <Text style={styles.textQuestion}>{constans.mainQuest}</Text>
         </View>
         <View style={styles.optionsUp}>
-          <ButtonMain text='Register Symptoms' img={require('../utils/images/symptoms.png')} onP={()=>{navigation.navigate('Login')}}/>
-          <ButtonMain text='Register Labs' img={require('../utils/images/Labs.png')} onP={()=>{navigation.navigate('Login')}}/>
-        </View>
-        <View style={styles.optionsDown}>
-          <ButtonMain text='See Locations' img={require('../utils/images/locations.png')} onP={()=>{navigation.navigate('Login')}}/>
+          <ButtonMain
+            text="Register Symptoms"
```
Comment (fernandoamz) [inline COVID19/src/screens/main.js:23]: The word `text` is a generical word you should change the variable's name

**Review by fernandoamz — CHANGES_REQUESTED** [review]

File: COVID19/src/utils/functions/mainFunctions.js (hunk 2 above)
Comment (hearagmu) [reply COVID19/src/utils/functions/mainFunctions.js:8]: The indicated code is precisely part of the getFixedDate function, is that what you mean?

File: COVID19/src/screens/main.js (hunk 7 above)
Comment (hearagmu) [reply COVID19/src/screens/main.js]: fixed

File: COVID19/src/screens/main.js (hunk 6 above)
Comment (hearagmu) [reply COVID19/src/screens/main.js]: fixed

File: COVID19/src/components/CustomCheck.js (hunk 4 above)
Comment (hearagmu) [reply COVID19/src/components/CustomCheck.js]: fixed

File: COVID19/src/utils/functions/mainFunctions.js (hunk 3 above)
Comment (hearagmu) [reply COVID19/src/utils/functions/mainFunctions.js]: fixed

File: COVID19/src/screens/registerSymptoms.js (hunk 1 above)
Comment (hearagmu) [reply COVID19/src/screens/registerSymptoms.js]: fixed

**Review by fernandoamz — APPROVED** [review]

**Review by mariam-rivera — APPROVED** [review]

**Review by jjcr11 — APPROVED** [review]
Good

**Pull request merged**